### PR TITLE
Fix use virtual servers

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Changes all ingresses to virtual servers

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
 - Users were not able to access ingress based services

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: .github/workflows/deploy-apis-to-production.yml [hunk 1]
```diff
@@ -1795,8 +1795,5 @@ jobs:
           method: kubeconfig
           kubeconfig: ${{ secrets.K8S_CONFIG_PROD }}
 
-      - name: Apply ingress changes
-        run: |
-          kubectl apply -f k8s/nginx/production/ingress.yaml
-          kubectl apply -f k8s/nginx/production/virtual-server.yaml
-          kubectl apply -f k8s/nginx/production/global-config.yaml
+      - name: Apply nginx changes
+        run: kubectl apply -f k8s/nginx/production/
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-production.yml:1799]: [update prod workflow to apply all files at once](https://github.com/airqo-platform/AirQo-api/pull/1526/commits/5f2b43ec20fdf3031c674c68ff2007d5a19618c6)

File: k8s/nginx/production/ingress.yaml
Comment (123MwanjeMike) [inline k8s/nginx/production/ingress.yaml:1]: [remove ingress](https://github.com/airqo-platform/AirQo-api/pull/1526/commits/3216bd4643133ac4ab41fc8ac91b342424fb5690)

File: k8s/nginx/production/airqalibrate-vs.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: k8s.nginx.org/v1
+kind: VirtualServer
+metadata:
+  name: airqalibrate-virtual-server
+  namespace: production
+spec:
+  host: airqalibrate.airqo.net
+  upstreams:
```
Comment (123MwanjeMike) [inline k8s/nginx/production/airqalibrate-vs.yaml:8]: [add airqalibrate-virtual-server](https://github.com/airqo-platform/AirQo-api/pull/1526/commits/092b911e13a31ffe26e88b0b26eb72b61446b4d3)

File: k8s/nginx/production/analytics-vs.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+apiVersion: k8s.nginx.org/v1
+kind: VirtualServer
+metadata:
+  name: analytics-virtual-server
+  namespace: production
+spec:
+  host: analytics.airqo.net
```
Comment (123MwanjeMike) [inline k8s/nginx/production/analytics-vs.yaml:7]: [add analytics-virtual-server](https://github.com/airqo-platform/AirQo-api/pull/1526/commits/db994eaef8ce97759a58b4ac0817930c7f7b0927)

File: k8s/nginx/production/docs-vs.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+apiVersion: k8s.nginx.org/v1
+kind: VirtualServer
+metadata:
+  name: docs-virtual-server
+  namespace: production
+spec:
+  host: docs.airqo.net
```
Comment (123MwanjeMike) [inline k8s/nginx/production/docs-vs.yaml:7]: [docs-virtual-server](https://github.com/airqo-platform/AirQo-api/pull/1526/commits/aae862e34c544d45c0e145f8aa192dc1557afb62)

File: k8s/nginx/production/platform-vs.yaml [hunk 5]
```diff
@@ -40,7 +40,7 @@ spec:
 apiVersion: k8s.nginx.org/v1
 kind: VirtualServer
 metadata:
-  name: production-virtual-server
+  name: platform-virtual-server
```
Comment (123MwanjeMike) [inline k8s/nginx/production/platform-vs.yaml:43]: [platform-virtual-server](https://github.com/airqo-platform/AirQo-api/pull/1526/commits/c214a829b13af7d7cf6ee2145d61c0ee1066d4c6)

File: .github/workflows/deploy-apis-to-staging.yml [hunk 6]
```diff
@@ -1802,8 +1802,5 @@ jobs:
           method: kubeconfig
           kubeconfig: ${{ secrets.K8S_CONFIG_STAGE }}
 
-      - name: Apply ingress changes
-        run: |
-          kubectl apply -f k8s/nginx/staging/ingress.yaml
-          kubectl apply -f k8s/nginx/staging/virtual-server.yaml
-          kubectl apply -f k8s/nginx/staging/global-config.yaml
+      - name: Apply nginx changes
+        run: kubectl apply -f k8s/nginx/staging/
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-staging.yml:1806]: [update staging workflow to apply all files at once](https://github.com/airqo-platform/AirQo-api/pull/1526/commits/c9c22af49604c96188a5f315f9931a99fa7770db)

File: k8s/nginx/staging/airqalibrate-vs.yaml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+apiVersion: k8s.nginx.org/v1
+kind: VirtualServer
+metadata:
+  name: staging-airqalibrate-virtual-server
+  namespace: staging
+spec:
+  host: staging-airqalibrate.airqo.net
```
Comment (123MwanjeMike) [inline k8s/nginx/staging/airqalibrate-vs.yaml:7]: [staging-airqalibrate-virtual-server](https://github.com/airqo-platform/AirQo-api/pull/1526/commits/dbe65203b03f465399c45bf51f91d243c3251034)

**Review by Codebmk — APPROVED** [review]

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**